<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-6J8TP4T" failures="1" tests="7" name="Test" time="0.351" errors="0" timestamp="2025-06-04T06:17:33 IST">
  <testcase classname="com.app.testng.FailedTestCase" name="test1" time="0.051"/>
  <testcase classname="com.app.testng.FailedTestCase" name="test2" time="0.072">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="com.app.testng.FailedTestCase" name="test2" time="0.003">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="com.app.testng.FailedTestCase" name="test2" time="0.005">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="com.app.testng.FailedTestCase" name="test2" time="0.002">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="com.app.testng.FailedTestCase" name="test2" time="0.005">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="com.app.testng.FailedTestCase" name="test2" time="0.004">
    <failure type="java.lang.AssertionError" message="expected [Ratna] but found [Evanglin Lavanya]">
      <![CDATA[java.lang.AssertionError: expected [Ratna] but found [Evanglin Lavanya]
at com.app.testng.FailedTestCase.test2(FailedTestCase.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
</testsuite> <!-- Test -->
